src/components/ButtonPanel.js

```diff
@@ -9,13 +9,19 @@ const ButtonPanel = () => {
     ['1', '2', '3', '+'],
     ['0', '.', '='],
   ];
-
+  const size = sign => sign === '0' && true;
```

Comment:
the `true` here is redundant, think about it, when `sign === '0'` is false, `false&&true` will be `false` and when `sign === '0'` is true, `true&&true` will be `true`